src/helpers/requiredServerFilesType.ts

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable eslint-comments/disable-enable-pair, @typescript-eslint/no-empty-interface, no-use-before-define */
```

Comment:
Ah, nice. Maybe we should contribute this upstream!

Reply:
Good call! Although I'm sure the definition file is missing some types since it was just auto-generated based on the demo's `required-server-files.json`